test/default_style.jl

```diff
@@ -2878,6 +2878,15 @@
             end
         end"""
         @test fmt(str, 4, 1) == str
+
+        # https://github.com/domluna/JuliaFormatter.jl/issues/453
+        str = """
+        bar = Dict(
+            :foo => \"""A triple quoted literal string
+                    with some words in it.\""",
+        )
+        """
+        @test fmt(fmt(str, 4, 20), 4, 20) == str
```

Comment:
how come this calls `fmt` twice?

Reply:
fmt already formats things twice btw, fmt1 is equivalent to calling format_text once

Reply:
ok, fixed